### PR TITLE
[ACS-8688] export playwright shared lib

### PR DESCRIPTION
**Pull request by akashrathod28**
**JIRA ticket link or change set's description**

export aca-playwright-shared lib to used in ADW playwright e2e for better code reuse
https://hyland.atlassian.net/browse/ACS-8688

### CODE REVIEW

**Review by DenysVuika — APPROVED** [review]

File: projects/aca-playwright-shared/project.json [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
+          "tsConfig": "projects/aca-playwright-shared/tsconfig.lib.prod.json"
+        }
+      },
+      "defaultConfiguration": "production",
+      "outputs": [
+        "{workspaceRoot}/dist/@alfresco/aca-playwright-shared"
+      ]
+    },
+    "lint": {
+      "executor": "@angular-eslint/builder:lint"
```
Comment (DenysVuika) [inline projects/aca-playwright-shared/project.json:25]: I think it's missing some configuration, the CI/CD says "nothing to lint"

Comment (akashrathod28) [reply projects/aca-playwright-shared/project.json:25]: Yes I miss to add .eslintrc.js file, will add it in next commit. 
seems like there any many lint issue after adding this on local to be resolved

File: tsconfig.adf.json [hunk 2]
```diff
@@ -36,7 +36,6 @@
       "@alfresco/aca-shared/store": ["projects/aca-shared/store/src/public-api.ts"],
       "@alfresco/aca-shared/rules": ["projects/aca-shared/rules/src/public-api.ts"],
       "@alfresco/aca-content/ms-office": ["projects/aca-content/ms-office/src/public-api.ts"],
-      "@alfresco/aca-testing-shared": ["projects/aca-testing-shared/src/index.ts"],
```
Comment (MichalKinas) [inline tsconfig.adf.json:39]: New library should be added here

File: projects/aca-playwright-shared/project.json [hunk 3]
```diff
@@ -1,7 +1,28 @@
 {
-  "name": "playwright-shared",
```
Comment (MichalKinas) [inline projects/aca-playwright-shared/project.json:2]: I think `ng-packge` file is missing

File: tsconfig.adf.json (hunk 2 above)
Comment (akashrathod28) [reply tsconfig.adf.json:39]: @alfresco/playwright-shared ... its present on line 24, so did not move that.

Comment (MichalKinas) [reply tsconfig.adf.json:39]: Ah okay didn't saw that

**Review by AleksanderSklorz — APPROVED** [review]

**Review by MichalKinas — APPROVED** [review]

**Pull request merged**